test/ffmpeg-qsv/encode/avc.py

```diff
@@ -151,3 +151,27 @@ def test(self, case, gop, slices, bitrate, fps, quality, refs, profile):
       slices    = slices,
     )
     self.encode()
+
+class vbr_lookahead(AVCEncoderTest):
+  @platform_tags(AVC_ENCODE_PLATFORMS)
+  @slash.requires(have_ffmpeg_h264_qsv_encode)
+  @slash.requires(have_ffmpeg_h264_qsv_decode)
+  @slash.parametrize(*gen_avc_vbr_la_parameters(spec, ['high', 'main', 'baseline']))
+  def test(self, case, bframes, bitrate, fps, quality, refs, profile, lookahead, lookaheaddepth):
+    vars(self).update(spec[case].copy())
+    vars(self).update(
+      bframes   = bframes,
+      bitrate   = bitrate,
+      case      = case,
+      fps       = fps,
+      lowpower  = 0,
+      maxrate   = bitrate * 2, # target percentage 50%
+      minrate   = bitrate,
+      profile   = profile,
+      quality   = quality,
+      rcmode    = "vbr",
+      refs      = refs,
+      lookahead = lookahead,
```

Comment:
`lookahead` not needed.  The presence of `la_depth` can be used to imply lookahead mode